.gitignore

```diff
@@ -172,3 +173,14 @@ Thumbs.db
 # Temporary file for logging scripts in xcode runs, see build-utils.sh for
 # details
 /xcodebuild-scripts.log
+
+# Linux packaging related
+/debian/
+results/
+*.deb
+*.ddeb
+*.dsc
+*.build
+*.buildinfo
+*.changes
+*.tar.?z
```

Comment:
These really should _not_ be in the top-level .gitignore. Can you move them to `linux/.gitignore`, or prefix as needed with `/linux/` please?

If it turns out that the Debian build is scattering files everywhere, let's try and fix that instead of using this big and dangerous hammer.

Reply:
Unfortunately these files do get generated in the root of the Keyman tree, so they'll have to live in this file. However, I limited the scope (PR #7451).
I also created an issue to try to refactor the Linux packaging (#7450).